packages/web3-common/src/errors.ts

```diff
@@ -398,14 +408,28 @@ export class AbiError extends Web3Error {
 
 export class PrivateKeyLengthError extends Web3Error {
 	public code = ERR_PRIVATE_KEY_LENGTH;
-	public constructor(value: string | Buffer) {
-		super(`Invalid value given "${value.toString()}". Error: Private key must be 32 bytes.`);
+	public constructor() {
+		super(`Error: Private key must be 32 bytes.`);
```

Comment:
```suggestion
		super(`Private key must be 32 bytes.`);
```

No need to have prefix `Error: ` JavaScript's native Error class will append it to the message